web/src/lib/search/typesForSearch.ts

```diff
@@ -1,5 +1,7 @@
 export type Match = {
   filename: string;
+  cmsCollectionName?: string; // TODO: remove ? to make required param
```

Comment:
[pebble] We had made this an optional param while we were proving the concept. Let's clean this up.
```
cmsCollectionName: string;
```